src/test/java/racingcar/ApplicationTest.java

```diff
@@ -31,6 +37,54 @@ class ApplicationTest extends NsTest {
         );
     }
 
+    @Test
+    void 예외_테스트_입력공백() {
+        assertSimpleTest(() ->
+                assertThatThrownBy(() -> runException("", "1"))
+                        .isInstanceOf(IllegalArgumentException.class)
+        );
+    }
```

Comment:
```suggestion
    @Test
    void 예외_테스트_입력공백() {
        assertSimpleTest(() ->
                assertThatThrownBy(() -> runException("", "1"))
                        .isInstanceOf(IllegalArgumentException.class)
                        .hasMessage("[ERROR] 입력이 비어있습니다")
        );
    }
```

hasMessage를 사용한다면 좀 더 정확한 검증이 될 것 같아요!

Reply:
감사합니다!! 다음 미션 때 참고할게요 ㅎㅎ